src/main/java/core/basesyntax/service/ConsoleServiceImpl.java

```diff
@@ -0,0 +1,209 @@
+                                    + splitData[1] + " "
+                                    + args[0];
+                    create(newCommand.split(" "));
+                } else {
+                    System.out.println("Неверный аргумент(ы). "
+                            + "Введите \"help\" для справки.");
+                }
+            }
+        }
+    }
+
+    @Override
+    public void exit() {
+        System.exit(0);
+    }
+
+    @Override
+    public void help(String... args) {
+        if (args.length == 2) {
+            String msg = "";
+            switch (args[1]) {
+                case "exit": {
+                    msg = String.format("%7s   %-25s   %-20s", args[1], "[code]",
+                            "Завершение работы программы.");
+                    break;
+                }
+                case "create": {
+                    msg = String.format("%7s   %-25s   %-20s", args[1], "[path] [file-name] [text]",
+                            "Создает текстовый файл по указанному пути.");
+                    break;
+                }
+                case "read": {
+                    msg = String.format("%7s   %-25s   %-20s", args[1], "[path] [file-name]",
+                            "Считывает файл по указанному пути" + " и выводит текст в консоль.");
+                    break;
+                }
+                case "info": {
+                    msg = String.format("%7s   %-25s   %-20s", args[1], "[path] [file-name]",
+                            "Выводит краткую информацию по указанному файлу.");
+                    break;
+                }
+                case "help": {
+                    msg = String.format("%7s   %-25s   %-20s", args[1], "[command]",
+                            "Выводит в консоль " + "все доступные комманды и информацию к ним.");
+                    break;
+                }
+                default: {
+                    System.out.println("Неверный аргумент : "
+                            + args[1] + ". \"help\" для справки.");
+                    break;
+                }
+            }
+            if (msg.length() > 0) {
+                System.out.println(msg);
+            }
+        }
+        if (args.length == 1) {
+            List<String> commands = List.of("create", "read", "info", "help", "exit");
+            for (String s : commands) {
+                help("help", s);
+            }
+        }
+    }
+
+    @Override
+    public void info(String... args) {
+        if (args.length < 3) {
+            System.out.println("Неверный аргумент(ы). Введите \"help\" для справки.");
+            return;
+        }
+
+        Path path = Paths.get(args[1] + File.separator + args[2]);
+
+        if (!Files.exists(Paths.get(args[1]))) {
+            System.out.println("Путь не существует.");
+            return;
+        }
+
+        if (!Files.exists(path)) {
+            System.out.println("Файла с таким именем не существует.");
+            return;
+        }
+
```

Comment:
we can move this block to separate method since we are using it in info and in read methods. For example: boolean checkInputArgs(String[] args)